design/arc/sequence_diagram_transactionDetails.mmd

```diff
@@ -0,0 +1,27 @@
+sequenceDiagram
+actor User
+participant FE_ARC
+participant BE_ARC
+participant Biz_Events_Service
+participant NDP
+
+note right of User: User authenticated
+note over User, NDP : Retrieve Transaction Details
+    User ->> FE_ARC : Access to transaction details
+    activate FE_ARC
+        FE_ARC ->> BE_ARC : GET Transaction Details/<transactionId> <br> Authentication: Bearer <Session Token>
+        activate BE_ARC
+            BE_ARC ->> BE_ARC : Validate session token
+            note right of BE_ARC : Retrieving CF from token
+            BE_ARC ->> Biz_Events_Service : Fiscal Code, Transaction Id
+            activate Biz_Events_Service
+                Biz_Events_Service ->> NDP : retrieve user transaction details from the "nodo-dei-pagamenti"
```

Comment:
not sure to model NDP in this diagram. I'm not even sure it is correct: BizEvents reads the transactions list from its cosmos collection and not directly from NDP

Reply:
look at [DR](https://pagopa.atlassian.net/wiki/spaces/PPR/pages/872218824/Design+Review+Lista+Transazioni#Vista-dinamica-delle-componenti) for more insights.

Reply:
I can't see the link you passed, so I remove the NDP participant in this and in the  transactions list diagram?

Reply:
yes please